RoadNamer/road-namer.json

```diff
@@ -0,0 +1,159 @@
+{  
+  "roadnames":[  
```

Comment:
I should get around to incorporating this into the US random name XML file or something